props/blaster_BC_buttons.h

```diff
@@ -0,0 +1,767 @@
+Reload                    - Click MODE.
+Fire                      - Click FIRE. (Hold to Auto / Rapid Fire when AUTO mode selected.)
+Clip In                   - Clip Detect pad Latched On.
+Clip out                  - Clip Detect pad Latched Off.
+Unjam                     - Bang the blaster or Reload, or Change Preset.
+
+=================================
+| 3 Buttons: FIRE, MODE, RELOAD |
+=================================
+*** Controls are the same as 2 button above, but with these changes ***
+
+Reload                    - Click RELOAD.
+Cycle Modes               - Click MODE.
+Change Preset             - Hold MODE
+        Next                  (while NOT pointing UP or DOWN).
+        Previous              (while pointing DOWN).
+        Jump to First         (while pointing UP).
+
+- If defined RELOAD_BUTTON_IS_POWER
+  Power ON/OFF              - Hold RELOAD until power ON/OFF.
+
+
+Wavs to use for talking Mode (else Talkie voice):
+- If these are not present, mode.wav will be used for all modes.
+- If no mode.wav either, then Talkie voice speaks selected mode.
+  mdstun.wav
+  mdkill.wav
+  mdauto.wav
+Additionally:
+  quote.wav      // for playing quotes
+  in.wav         // for blaster power off
+  out.wav        // for blaster power on
+  destruct.wav   // for self destruct overload
+  boom.wav       // for self destruct explosion
+These can be used to match a more "plasmatic" stun sound.
+  clipins.wav
+  clipouts.wav
+  reloads.wav
+
+The following are likely best kept in a 'common' folder in your Font Search Path, but of course
+can be specific versions within a font as well.
+Choose a Voice Pack containing all these sounds here: https://fredrik.hubbe.net/lightsaber/sound/
+
+  Battery level wavs:
+    mzero.wav
+    mnum 1-20 wavs, as well as the "tens" thirty.wav - ninety.wav
+    battery.wav    // for accompanying EFFECT_BATTERY_LEVEL
+    battlevl.wav   // for Spoken Battery Level
+    mpercent.wav
+    mpoint.wav
+    mvolts.wav
+  vmbegin.wav    // for Begin Volume Menu
+  vmend.wav      // for End Volume Menu
+  volup.wav      // for increse volume
+  voldown.wav    // for decrease volume
+  volmin.wav     // for minimum volume reached
+  volmax.wav     // for maximum volume reached
+  mfalse.wav     // "disabled" used for sequential quote mode
+  mtrue.wav      // "enabled" used for sequential quote mode
+  mrandom.wav    // "ramdom" used for sequential quote mode
+*/
+
+#ifndef PROPS_BLASTER_BC_BUTTONS_H
+#define PROPS_BLASTER_BC_BUTTONS_H
+
+#ifndef BLASTER_DEFAULT_MODE
+#define BLASTER_DEFAULT_MODE MODE_KILL
+#endif
+
+#include "blaster.h"
+#include "../sound/sound_library.h"
+#include "../modes/settings_menues.h"
+
+#ifdef PROP_TYPE
+#undef PROP_TYPE
+#endif
+
+#define PROP_TYPE BlasterBCButtons
+
+#ifndef BUTTON_HELD_LONG_TIMEOUT
+#define BUTTON_HELD_LONG_TIMEOUT 1200
+#endif
+
+// Additional effects for BC buttons prop.
+EFFECT(clipins);    // s for stun mode
```

Comment:
Seems like it would be better to use alt modes for stun/kill/auto, that way *every* effect can be individualized if you want to, without adding any more code.

Reply:
Sorry, I guess I don't know what you mean by every effect could be individualized.
Unless you mean to have 3 sets of clipin, clipout, and reload in the font, then call the appropriate alt for each mode kill, stun, auto?
Again, would that be in the prop? I thought alt sounds were "meant" to be changed via blade style code.

Reply:
https://pod.hubbe.net/sound/alt_sounds.html#have-the-prop-control-the-alt

Reply:
> Seems like it would be better to use alt modes for stun/kill/auto, that way _every_ effect can be individualized if you want to, without adding any more code.

Done.